frontend/src/e2e-test/specs/5_employee/assistance-need-decision.spec.ts

```diff
@@ -115,6 +116,27 @@ describe('Assistance Need Decisions - Edit page', () => {
     )
     await assistanceNeedDecisionEditPage.waitUntilSaved()
   })
+
+  test('Clicking the preview button opens the decision in preview mode', async () => {
+    await assistanceNeedDecisionEditPage.assertDeciderSelectVisible()
+    await assistanceNeedDecisionEditPage.clickPreviewButton()
+
+    assertEquals(
```

Comment:
Could these `assertEquals` use Jest's built-in `expect(…).isEqual(…)`? (also below)